ming_sim/action_materialize.py

```
@@ -2781,17 +2886,23 @@ def _materialize_appointment(ctx: MaterializeCtx) -> None:
             )
             return
         if status == "hit" and row is not None:
-            # 同人同职再发任命+路径：no-op 去重，中旨/任别并入既有条
-            pending_id = _annotate_office_pending_path(
-                session.db,
+            # 同人同职再发任命+路径：no-op 去重，中旨/任别/summon 并入既有条
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)